### ISSUE TITLE
Updating OCDS Maturity Model

### ISSUE THREAD

**Issue by timgdavies** [open-contracting/standard#388]
## The issue

OCDS comes with [detailed implementation guidance](http://standard.open-contracting.org/latest/en/implementation/) and a 'Basic', 'Intermediate' and 'Advanced' framework. 

Publishers want to know which Basic, Intermediate or Advanced (BIA) level they are at, and which star-level they are at. This cannot be easily assessed based on current documentation because:
- BIA mixes the issue of which stages are published, with issues of the specific fields of data that have been published (e.g. measures both whether you are published tender, award and contract, and what is being published at each stage)

Depending on the systems a publisher has, it can be easy to publish advanced fields, and difficult to provide some basic fields.

There are also issues with the star system for assessing data quality, as some publishers feel that the star levels are not clearly defined and lack examples, and that the levels are not contextual.

We need a revised ‘maturity model’ to help publishers plan, publish and improve

We need to clarify whether there can be objective assessments, or only self-assessment, and to develop tooling to support our revised approach to assessment.
## Proposed changes

We propose to updating the documentation to clarify the key areas for disclosure in a contracting process and to develop a maturity model that encourages publishers to improve the depth of information on key themes such as:
- What is being procured
- When is it being procured
- How is it being procured
- How much is being spent
- Who is procuring
- Who is buying
- Who is supplying

As #214 explores, we should also highlight the importance of providing documents and should aim to separate our assessment of document disclosure from BIA level and to separately detail the documents required for each stage.
## Discussion

We will look to work up detailed proposal on an updated maturing model in the coming months.

We invite feedback on the current model, and views on whether an updated maturity model should form part of OCDS 1.1, or be developed separately.

**Comment by mireille-raad** [comment]
Here is some feedback on this issue from **Abdoulaye Fabregas** from the OC team at WB:

Some issues arising from the BIA model, for example the fact that most countries will have mixed features from B, I and A level. This makes it difficult to use the BIA model as a benchmark, and at the same time it can disincentivize countries from moving forward once they consider themselves “compliant” with the basic level. On the other hand, the categorization in different levels (vs a monolithic OCDS) gives countries flexibility to prioritize and use a phased approach to the implementation of the Standard.

As an alternative, and since use cases are the basis of our country engagement (and of the OC approach I think), wouldn’t it make sense to replace the BIA categorization by the four OCDS use cases:
1/ Achieving Value for Money;
2/ Detecting fraud and corruption;
3/ Competing for public contracts;
4/ Monitoring service delivery.

I feel this would still allow countries to prioritize their disclosure, but the choice of the data, information and documents that they disclose would be driven by the objective that each country is trying to achieve.

**Comment by timgdavies** [comment]
Thanks @dc-coder for the feedback from Abdoulaye.

We also discussed this on today's community call. 

Eliza Niewiadomska emphasised the importance of the publication levels for helping countries translate commitments to transparency into actual practice.

We discussed the extent to which requirements to public are determined by national law, and trade agreements - and the extent to which we might be able to map fields to particular requirements there.

We will explore more options to develop a more responsive approach to the publication levels.

**Comment by timgdavies** [comment]
Work on this is ongoing. Moved to post 1.1.

**Comment by jpmckinney** [comment]
The specific page this issue is about is http://standard.open-contracting.org/latest/en/implementation/levels/

**Comment by juanpane** [comment]
Currently there are two places in the documentation saying how to implement the 5 star approach

- http://standard.open-contracting.org/latest/en/implementation/levels/#how-to-publish-5-approach, and
- http://standard.open-contracting.org/latest/en/getting_started/publication_patterns/ .

If a publisher implements OCDS and publishes an API with an URI for their data, according to the first documentation it might seem that they reach 4 stars, however, considering the second documentation page it might seem they don't.

The big difference seems to be the 3rd star where for the first documentation seems desirable to have a bulk data download, but in the second documentation it seems that it is mandatory.

Can we please check this and have only one definition, a single source of truth, to avoid confusion in the publishers?

**Comment by jpmckinney** [comment]
Indeed. A quick solution may be to simply remove the publication patterns page (or at minimum its star levels content): the star levels repeat what's on the levels page; packaging should likely go on the releases and records page; and bulk files should likely go on the [data files](http://standard.open-contracting.org/latest/en/implementation/hosting/) page, as raised in #605.

**Comment by jpmckinney** [comment]
Noting that the stars are mentioned on many pages, but can be easily found by searching for `☆`.

**Comment by jpmckinney** [comment]
#980 replaces references to ☆ levels (solving @juanpane's issue above), but retains the levels page itself, until we have a suitable replacement.

**Comment by LindseyAM** [comment]
We drafted some text that revises 'publication levels' to give guidance on improving OCDS quality and completeness more in line with OCP's strategic targets for publication & improvement while still giving some 'advanced' criteria to the field  that emphasizes the usability of the data and documents: https://docs.google.com/document/d/1rzcLldNeWTtfW34n_WrIc3B-WY3D2D61wFBN7_fiCnU/edit#

**Comment by timgdavies** [comment]
I support the proposal to update the BIA model, although I find the current draft text fairly hard to parse, and assuming a very technical audience - when part of the challenge with use of BIA was that policy makers want to understand how well their publication is performing against a benchmark. 

I would suggest leading with the **why** before the **how** in each of the bullets.

E.g. 

* Completeness
  * Publish multiple releases per OCID -> **Provide data each time something significant happens in the contracting process by** publishing multiple releases for each `OCID` (process)

**Comment by LindseyAM** [comment]
Thank you very much to Tim, Jiri, Martin & Dmitry for the helpful feedback and suggestions. The draft has been updated to talk more about why these checks are relevant, how we work with publishers to document improvements, and other requested clarifications.
